core/src/apps/cardano/layout.py

```diff
@@ -425,6 +425,16 @@ async def confirm_catalyst_registration(
     )
 
 
+async def confirm_script_data_hash(ctx: wire.Context, script_data_hash: bytes) -> None:
+    await confirm_properties(
+        ctx,
+        "confirm_script_data_hash",
+        title="Confirm transaction",
+        props=[("Script data hash:", script_data_hash)],
```

Comment:
Please add a TODO comment here so we don't forget to decide if the hash shouldn't be shown in bech32.